ironfish/src/wallet/wallet.test.ts

```diff
@@ -1040,6 +1040,53 @@ describe('Accounts', () => {
 
       expect(spentNoteHashes.has(notes[0])).toBe(true)
     })
+
+    it('should create transactions with spends valid after a reorg', async () => {
```

Comment:
Should this be `invalid`? Or are you going to use this to test changes to transaction creation?

Reply:
Yeah, it is invalid now, but I'm hoping that with changes it will become valid